react-app/src/app/screens/Game/components/Moves/components/Move/index.js

```diff
@@ -1,33 +1,32 @@
-import React, { Component, Fragment } from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
-import { jumpTo } from '../../../../../redux/Game/actions';
-import calculateWinner from '../../utils';
+import { jumpTo } from '../../../../../../../redux/Game/actions';
+import calculateWinner from '../../../../utils';
 
 import style from './styles.scss';
 
 class Move extends Component {
-  createHistoryButton(move) {
-    const desc = move ? `Go to move # ${move}` : 'Go to game start';
-    const history = this.props.history.slice(0, move + 1);
+  render() {
+    const desc = this.props.move ? `Go to move # ${this.props.move}` : 'Go to game start';
+    const history = this.props.history.slice(0, this.props.move + 1);
     const current = history[history.length - 1];
     const squares = current.squares.asMutable().slice();
+
     return (
       <li>
         <button
           className={style.move}
-          onClick={() => this.props.jumpTo(move, move % 2 === 0, calculateWinner(squares))}
+          onClick={() => {
+            this.props.jumpTo(this.props.move, this.props.move % 2 === 0, calculateWinner(squares));
```

Comment:
Avoid defining new function references as props in the render method. Instead, extract this function call to a method and pass its reference to the onClick prop.
More info here: https://medium.freecodecamp.org/why-arrow-functions-and-bind-in-reacts-render-are-problematic-f1c08b060e36